libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -914,16 +1132,31 @@
         "comment": "",
         "limit": 0
     },
+    "dashboard.bucket.measures_title.bullet|metric": {
+        "value": "Metric",
+        "comment": "",
+        "limit": 0
+    },
     "dashboard.bucket.secondary_measures_title.bullet": {
         "value": "Measure",
         "comment": "",
         "limit": 0
     },
+    "dashboard.bucket.secondary_measures_title.bullet|metric": {
+        "value": "Metric",
+        "comment": "",
+        "limit": 0
+    },
     "dashboard.bucket.tertiary_measures_title.bullet": {
         "value": "Measure",
         "comment": "",
         "limit": 0
     },
+    "dashboard.bucket.tertiary_measures_title.bullet|metric": {
+        "value": "Metric",
+        "comment": "",
+        "limit": 0
+    },
     "dashboard.bucket.measures_subtitle.bullet": {
         "value": "Primary",
         "comment": "Primary measure of a bullet chart.",
```

Comment:
There are texts with `measure` in [libs/sdk-ui/src/base/localization/bundles/en-US.json](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/bundles/en-US.json#L63) and  [libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json) as well, for example:
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui/src/base/localization/bundles/en-US.json#L63
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui/src/base/localization/bundles/en-US.json#L68
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json#L637
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json#L642

→ Please don't forget to add new texts containing `metric` in those 2 files.

Reply:
fixed